### DISCUSSION TITLE
Thank you for the Gnome Openweather Fork!

### DISCUSSION THREAD

**Discussion by GerryT11** [penguin-teal/gnome-openweather#27] [General]
Hi, I just wanted to say Thank you for Gnome Openweather. I have been using Openweather for many years on Gnome < 45. Just today, I came across this fork and I like it very much. It works smooth and looks nice.

**Comment by penguin-teal** [comment]
You're welcome! I too used the original extension and was surprised to see it go, especially considering how popular it was.
